### PR TITLE
Upgrade  css-loader to 5.1.3

### LINKED ISSUE #4815: Bump css-loader from 3.6.0 to 5.x.x #4765
IN #4765 Dependabot tried to upgrade our version of css-loader to 5.1.1

Unfortunately, this upgrade breaks the loading of static image and font assets which now resolved differently.

* Expected: 

- [ ] css-loader is upgraded to the latest version (5.x.x)
- [ ] make css works without error
- [ ] make js works without error
- [ ] static asset URLs are not resolved - we do not want to embed fonts and images inside the stylesheets. The `url` option of  https://www.npmjs.com/package/css-loader is likely needed.

### Stakeholders
@jdlrobson

### PR DESCRIPTION
**Pull request by jonbos**
My first ever PR to an open source project! 😀 thanks all.

Closes #4815 

**Fix** - bump CSS loader to latest and repair css-loader resolution via options in webpack.config

### Technical

Changes `--parent` flag to OSX friendly `-p` flag in both `make css` and `make js` rules

### Testing

Run
`$ make css`
`$ make js`

### Stakeholders

@jdlrobson

### CODE REVIEW

**Comment by jdlrobson** [comment]
Congrats on the milestone!!!! I will make sure I review by end of week but this looks very promising! :)

**Comment by jonbos** [comment]
Thank you for the kind words! I love the Internet Archive and I'm eager to get further involved with this project.

**Comment by jdlrobson** [comment]
It looks like this leads to an increase in the bundlesize that's being flagged when `npm test` is run. Can you investigate whether that's expected?

If it is, we'll need to update the `bundlesize.config.json` definition with the new sizes. If not (for example, the stylesheet might now base64 encode assets where we don't want to it), we may want to consider tweaking the solution.

**Comment by jonbos** [comment]
@jdlrobson I noticed that too - I wasn't sure whether or not the size increase was expected. I will look into it!

**Comment by jdlrobson** [comment]
i ran `make js` on current master branch and then again on your branch with `webpackconfig.optimization.minimize = false;`. When I review the diff of the two files I am seeing changes to CSS in the webpack bundle but nothing that concerns data URIs or background images. I've also tested it and the patch works for me.

So I think it's acceptable to rebase this patch on master and update the bundlesize like so https://github.com/internetarchive/openlibrary/pull/4813/files#diff-b300d4a00ffb54905382b1328c39daa504b2f3be828103b372f8b5f6b8f796a2

I don't think I can push to your pull request, otherwise I'd do this myself and merge.

Thanks for the contribution!

File: package.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
     "chart.js": "2.9.4",
     "chartjs-plugin-datalabels": "0.7.0",
     "core-js": "3.9.1",
-    "css-loader": "3.6.0",
+    "css-loader": "^5.1.3",
```
Comment (jdlrobson) [inline package.json:37]: When rebasing please fix the version here and drop the ^. Using ^ means its harder for developers to reproduce the local environment, it also opens you up to security vulnerabilities by pulling down code from external places that has not been audited by us.

File: Readme.md [hunk 2]
```diff
@@ -36,8 +36,6 @@ Here's a quick public tour of Open Library to get your familiar with the service
 
 The development environment can be set up using the [Docker Instructions](https://github.com/internetarchive/openlibrary/blob/master/docker/README.md). You can also watch the [video tutorial](https://archive.org/embed/openlibrary-developer-docs/openlibrary-docker-set-up.mp4) for a more detailed explanation.
 
-Our `Docker` environment is in active development. Want to contribute? Here's our top-level [`Docker` todo-list](https://github.com/internetarchive/openlibrary/issues/1067) and a [list of open `Docker` issues](https://github.com/internetarchive/openlibrary/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Adocker).
-
 ### Developer's Guide
```
Comment (jdlrobson) [inline Readme.md:39]: There's some unrelated changes in this rebase.

**Review by jdlrobson — CHANGES_REQUESTED** [review]
I think something went wrong with the rebase here.  You may need to do a force push. It may be easier to create a new branch/pull request.

**Comment by jonbos** [comment]
New PR at [4921](https://github.com/internetarchive/openlibrary/pull/4921)

**Pull request closed without merging**